.github/workflows/docker.yml

```diff
@@ -0,0 +1,47 @@
+name: CI-Docker
```

Comment:
initially thought let's keep this labelled `CI` with job name `docker-test-dev-alpine`
but in hindsight current naming makes sense being our custom image